doc/source/whatsnew/v0.20.0.txt

```diff
@@ -416,6 +416,7 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug in ``Timestamp.replace`` wrong exception raised when wrong argument name given (:issue:`15240`)
```

Comment:
done and left in bug fixes as it is a bug introduced by a previous change (not really an excepted API change)